Public/Src/Tools/Execution.Analyzer/Analyzers.Core/Xldb/XLGToDBAnalyzer.cs

```diff
@@ -708,7 +720,11 @@ private void IngestProducerConsumerInformation()
                 var fileConsumerValue = new FileConsumerValue();
                 fileConsumerValue.PipIds.AddRange(kvp.Value);
 
-                WriteToDb(fileConsumerKey.ToByteArray(), fileConsumerValue.ToByteArray(), XldbDataStore.StaticGraphColumnFamilyName);
+                var key = fileConsumerKey.ToByteArray();
+                var value = fileConsumerValue.ToByteArray();
+                WriteToDb(key, value, XldbDataStore.StaticGraphColumnFamilyName);
+
+                AddToDbStorageDictionary(DBStoredTypes.FileConsumer, key.Length + value.Length);

```

Comment:
>AddToDbStorageDictionary(DBStoredTypes.FileConsumer, key.Length + value.Length); [](start = 16, length = 80)

here you are adding the size of both key and the value; in other places you are only adding the size of the value. we need to be consistent

Reply:
Adding key and value everywhere now.

---
In reply to: [321893027](https://github.com/microsoft/BuildXL/pull/836#discussion_r321893027) [](ancestors = 321893027)

Reply:
I see no changes around this

---
In reply to: [321949494](https://github.com/microsoft/BuildXL/pull/836#discussion_r321949494) [](ancestors = 321949494,321893027)